resources/views/pages/products.ejs

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <% include ../partials/head %>
+    <link rel="stylesheet" href="/resources/css/products.css" />
+  </head>
+  <body class="container">
+    <header>
+      <% include ../partials/header %>
+    </header>
+    <main class="flex-row">
+      <div class="category-list">
+        <% for(var i=0; i < categories.length; i++) { %>
+        <a href="/products/<%= categories[i].key %>"><h1 class="category-list__name"><%= categories[i].name %></h1></a>
+        <% } %>
+      </div>
+      <div class="flex-column product-list">
+        <div class="mob-category-list-container">
+        <select class="mob-category-list" onchange="window.location = this.value;">
```

Comment:
Event Binding should be in JS. Don't use onchange

Reply:
done